rules/providers.bzl

```diff
@@ -63,5 +68,9 @@ ExperimentalMetadataInfo = provider(
         "type": "string: How to interpret data",
         "label": "Label: label of the metadata rule",
         "data": "String->any: Map of names to values",
-    }
+    },
 )
+
+# Deprecated: Use write_licenses_info instead.
+# TODO: Remove once downstream customers are migrated off TransitiveLicensesInfo .
```

Comment:
TODO(version 0.0.6): Drop this.

We can't make an indefinite promise that people have migrated off.
At the 0.0.x level, we are allowed breaking changes at any release.
The 0.1.0 is certainly not going to try to be fully backwards compatible with 0.0.whatever

Reply:
Dropped.